resolution: 0.06
gate_off_ground: 0.21
gate_side_margin: 0.1